### ISSUE TITLE
Upgrade oauth2-proxy chart

### ISSUE THREAD

**Issue by j-zimnowoda** [redkubes/otomi-core#1126]
**Is your feature request related to a problem? Please describe.**

From: 
```
chartVersion: v7.1.3
appVersion: 2.4.1      
```
To:
```
chartVersion: 7.4.0
appVersion: 6.12.0
```

See: https://artifacthub.io/packages/helm/oauth2-proxy/oauth2-proxy

**Describe the solution you'd like** A clear and concise description of what you want to happen.

**Describe alternatives you've considered** A clear and concise description of any alternative solutions or features
you've considered.

**Additional context** Add any other context or screenshots about the feature request here.

**Comment by Ani1357** [comment]
Updated oauth2-proxy chart to version 3.7.4. AppVersion = 7.4.0.
As part of the major update to version 3.*, the redis subchart is now added to the oauth2-proxy chart. As such I'm disabling the current redis and using adapting the go template for the new redis subchart.
Updating in place turned out to be quite problematic so I updated the upgrade script to delete the oauth2-proxy and oauth2-proxy-redis releases before upgrading to the new version. Created PR
